### DISCUSSION TITLE
I have read the remote setup instructions but deployment fails

### DISCUSSION THREAD

**Discussion by Lovenoreus** [microsoft/call-center-ai#361] [Q&A]
What do you need to add to the light config file for remote deployment?

I only realized long after trying it around that you are supposed to fill it out with more information than the phonenumbers but I have not created Azure chatbots before and don't know what is specifically required to add for remote deploment.

Can you give a finished example? The config.yaml for the local deployment has a different command and therefore it's content should not be the same.

I assumed I had to do step 1,2,3 in Prerequisites first and then use those resources for the "make deploy" but when I did the command with your base empty config.yaml it fails and says that the resource group already exists. If I do it without a previous resource group I just get these errors with a new normal azure:

![image](https://github.com/user-attachments/assets/c939b118-7d66-4d53-93e6-8ed33c6978ab)
![image](https://github.com/user-attachments/assets/2b3ad212-6130-4523-8d22-572cf00bdb89)

**Comment by clemlesne** [comment]
Hello,

The error explicitly says: `Resource has invalid content filter: Policy does not have necessary permission to override base policy. Please check aka.ms/oai/rai/exceptions`.

There is a flag named `prompt_content_filter` to enable or disable the prompt content filter rule. Disable the content filter rule requires permissions from Microsoft. It seems I inversed the flag, my fault, I’ll fix that.

In the meantime, ca you try to disable it? Just add `prompt_content_filter=false` at the end of your `make` command.
